### PR TITLE
Feat/tool buttons

### PR DESCRIPTION
**Pull request by vvandijck**
This pull request includes:
- Separate buttons for each tool, configurable using the tool's .json config file.
- Added an export button on step 3, to export the generated WMS layers as `.tiff` images in a single `.zip` file.

### CODE REVIEW

File: public/data/abstraction.json [hunk 1]
```diff
@@ -9,13 +9,23 @@
       "components": [
         "marker-list",
         "layers-graph"
-      ]
+      ],
+      "buttons": []
```
Comment (sjoerdbeentjes) [inline public/data/abstraction.json]: What do you think of calling this 'actions'? It's a bit less vague than 'buttons'? If we decide on that, let's also rename the `step-buttons` accordingly.

**Review by sjoerdbeentjes — CHANGES_REQUESTED** [review]
Nice way of solving this problem! Added one suggestion

File: public/data/abstraction.json (hunk 1 above)
Comment (vvandijck) [reply public/data/abstraction.json]: Good one 👍🏼

File: public/data/abstraction.json [hunk 2]
```diff
@@ -9,13 +9,23 @@
...
         "marker-configuration"
+      ],
+      "actions": [
+        "reset-button"
+      ]
+    },
+    {
+      "actions": [
+        "reset-button",
+        "save-button"
```
Comment (sjoerdbeentjes) [inline public/data/abstraction.json]: The 'button' part in the items now seems a bit unnecessary.

```suggestion
        "reset",
        "save"
```

**Review by sjoerdbeentjes — CHANGES_REQUESTED** [review]

File: public/data/abstraction.json (hunk 2 above)
Comment (vvandijck) [reply public/data/abstraction.json]: Changed!

**Review by sjoerdbeentjes — APPROVED** [review]

**Pull request merged**